OpenRA.Game/Orders/UnitOrderGenerator.cs

```diff
@@ -17,6 +17,9 @@ namespace OpenRA.Orders
 {
 	class UnitOrderGenerator : IOrderGenerator
 	{
+		private const string SelectCursor = "select";
```

Comment:
My point is that you can pass these in through the ctor (using default values) via traits that create this.

That is how you can read the values in from YAML.